iavl/utils.py

```diff
@@ -75,6 +75,9 @@ def hash(self):
     def encode(self):
         return encode_node(self)
 
+    def calc_balance(self, ndb):
```

Comment:
maybe can rm [iavl](https://github.com/crypto-com/python-iavl/blob/467f64c2305c2bbc6a1cf3ab12158e12f280fe27/iavl/iavl.py#L197) one then

Reply:
Both are needed, `PersistedNode` and `Node` can coexist in the same tree.

Reply:
you is right